keps/sig-network/2433-topology-aware-hints/README.md

```diff
@@ -307,9 +309,14 @@ had not yet propagated to all of them.
 ### EndpointSlice Controller
 
 When the `TopologyAwareHints` feature gate is enabled and the annotation is set
```

Comment:
this will make even harder to remove the annotation in the future ... and someone will want some more esoteric configuration

Reply:
Unfortunately I don't think the "Auto" approach ever really had a viable path to being safely on by default unless we found some kind of way to implement a feedback loop to understand how much load each endpoint had. I think we were going to be stuck with either an annotation or field to configure this with any approach we took.

Reply:
Although I agree that keeping an annotation forever is not ideal, I think it's mostly reasonable here. This is really just a hint for how you would like the underlying dataplane to behave.